gulpfile.js

```diff
@@ -115,6 +115,13 @@ gulp.task('build-dist-js', function() {
     .pipe(gulp.dest('dist'));
 });
 
+gulp.task('build-dist-scss', function() {
+  gulp.src('./example/css/react-calendar.scss')
```

Comment:
:question: Do you think it would make more sense to put this file into `src` now?